package.json

```diff
@@ -133,7 +133,7 @@
     "os-locale": "^4.0.0",
     "pkg-up": "^3.1.0",
     "portfinder": "^1.0.20",
-    "puppeteer-core": "^1.17.0",
+    "puppeteer-core": "^1.18.0",
```

Comment:
Puppeteer v1.18.0 has an updated protocol to create PDF. It will be supported from Chromium 77, and we cannot use for the current stable Chrome (75). Thus, we should stop to update Puppeteer for a while.

https://github.com/GoogleChrome/puppeteer/issues/4609